config/global.ini.php

```diff
@@ -11,6 +11,9 @@
 ; WARNING - YOU SHOULD NOT EDIT THIS FILE DIRECTLY - Edit config.ini.php instead.
 ;--------
 
+[path]
+tmp = "/tmp"
```

Comment:
* Would move this under `[General]`
* Can you add a comment explaining this new setting